### PR TITLE
Bur 44 frontend create responsive footer

### PR DESCRIPTION
**Pull request by driesvanspauwen**

### CODE REVIEW

File: frontend/src/app/(header-footer)/testtest/page.tsx
Comment (rserry) [inline frontend/src/app/(header-footer)/testtest/page.tsx:1]: maybe remove this page xD

File: frontend/src/app/layout.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import "./globals.css";
 import React from "react";
 import { Inter } from 'next/font/google'
 import CookieBanner from "@/components/cookie-banner/CookieBanner";
+import Footer from "@/components/footer/Footer";
```
Comment (rserry) [inline frontend/src/app/layout.tsx:5]: not used

File: frontend/src/components/footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                        <p className={styles.footerText}>{" or "}</p>
+                        <a href="https://github.com/VTKLeuven/burgieclan" className={styles.footerLink}>
+                            Contribute on Github<span><ArrowTopRightOnSquareIcon className="h-3 w-3 inline-block ml-0.5 mb-1 text-gray-500" aria-hidden="true" /></span>
+                        </a>
+                    </div>
+
+                    {/* Copyright Section */}
+                    <div>
+                        <p className="mt-2 text-xs leading-5 text-gray-500 md:order-1 md:mt-0">
+                            &copy; 2024 Vlaamse Technische Kring vzw
```
Comment (rserry) [inline frontend/src/components/footer/Footer.tsx:90]: Can you auto set current year?

File: frontend/src/components/header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                <div className="hidden md:flex md:gap-x-8">
+                    {navigation.map((item) => (
+                        <a key={item.name} href={item.href} className="text-sm font-semibold leading-6 text-gray-900">
+                            {item.name}
+                        </a>
+                    ))}
+                    {/* Mobile menu toggle button */}
+                    {/*TODO: move logic to global state (redux or react context)*/}
+                    {isAuthenticated ?
+                        <a href="profile" className="text-sm font-semibold leading-6 text-gray-900">
```
Comment (rserry) [inline frontend/src/components/header/Header.tsx:80]: redirect to /profile or /account?

File: frontend/src/components/header/Header.tsx [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+                </div>
+
+                {/* Whitespace and login/profile links */}
+                <div className="hidden md:flex md:gap-x-8">
+                    {navigation.map((item) => (
+                        <a key={item.name} href={item.href} className="text-sm font-semibold leading-6 text-gray-900">
+                            {item.name}
+                        </a>
+                    ))}
+                    {/* Mobile menu toggle button */}
```
Comment (rserry) [inline frontend/src/components/header/Header.tsx:77]: This comment is not needed here I think?

File: frontend/src/components/header/Header.tsx [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+                                        href={item.href}
+                                        className="-mx-3 block rounded-lg px-3 py-2 text-base font-semibold leading-7 text-gray-900 hover:bg-gray-50"
+                                    >
+                                        {item.name}
+                                    </a>
+                                ))}
+                            </div>
+                            <div className="py-6">
+                                <a
+                                    href="login"
```
Comment (rserry) [inline frontend/src/components/header/Header.tsx:134]: No check if user is already authenticated, like the desktop version

File: frontend/tsconfig.json [hunk 6]
```diff
@@ -24,4 +24,4 @@
   },
   "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
   "exclude": ["node_modules"]
-}
+}
```
Comment (rserry) [inline frontend/tsconfig.json:27]: why remove newline?

**Review by rserry — CHANGES_REQUESTED** [review]

File: frontend/src/components/header/Header.tsx (hunk 3 above)
Comment (rserry) [reply frontend/src/components/header/Header.tsx:80]: added to PR #108

File: frontend/src/components/header/Header.tsx (hunk 5 above)
Comment (rserry) [reply frontend/src/components/header/Header.tsx:134]: added to PR #108

File: frontend/src/components/header/Header.tsx (hunk 4 above)
Comment (rserry) [reply frontend/src/components/header/Header.tsx:77]: added to PR #108

**Comment by driesvanspauwen** [comment]
PR comments resolved and waiting for BUR-168 to release

**Review by rserry — APPROVED** [review]

**Pull request merged**